Remove empty directories from git

Presently in the PCD 1.1.6 SDK, there are empty directories (include & bin) in the PCD_ROOT. Move the creation of such directories to the Makefile.

Without the 'include' directory, the make fails & gives the following error:
```
$ make
*--------------------------------------------------
*  Process Control Daemon
*  Build date: Sun, 08 Oct 2017 09:22:54 +0530 :
*--------------------------------------------------
Checking write permission: /home/rp/pcd/include
Checking write permission: /home/rp/pcd/sbin/
Checking write permission: /home/rp/pcd
Building PCD...
make[1]: Entering directory '/home/rp/github/pcd/ipc/src'
  CC [C] 	ipc.o
  LINK	 	libipc
install: target ‘/home/rp/github/pcd/include’ is not a directory
Makefile:37: recipe for target 'libipc' failed
make[1]: *** [libipc] Error 1
make[1]: Leaving directory '/home/rp/github/pcd/ipc/src'
Makefile:141: recipe for target 'pcd' failed
make: *** [pcd] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -129,6 +129,8 @@ check_permissions:
 	@mkdir -p $(CONFIG_PCD_INSTALL_DIR_HOST) && touch $(CONFIG_PCD_INSTALL_DIR_HOST)/pcd.tmp 2> /dev/null && rm $(CONFIG_PCD_INSTALL_DIR_HOST)/pcd.tmp
 	@echo Checking write permission: $(CONFIG_PCD_INSTALL_DIR_PREFIX)
 	@mkdir -p $(CONFIG_PCD_INSTALL_DIR_PREFIX) && touch $(CONFIG_PCD_INSTALL_DIR_PREFIX)/pcd.tmp 2> /dev/null && rm $(CONFIG_PCD_INSTALL_DIR_PREFIX)/pcd.tmp
+	@echo Checking write permission: $(PCD_ROOT)/include
+	@mkdir -p $(PCD_ROOT)/include && touch $(PCD_ROOT)/include/pcd.tmp 2> /dev/null && rm $(PCD_ROOT)/include/pcd.tmp
 
 check_config:
 	@if [ ! -f $(PCD_ROOT)/.config ]; then \
@@ -167,5 +169,6 @@ clean:
 distclean: clean
 	@rm -f .config .config.old $(PCD_CFG_DIR)/pcd_autoconf.h $(PCD_CFG_DIR)/auto.conf
 	@rm -f $(PCD_KCFG_DIR)/.config $(PCD_KCFG_DIR)/.config.old $(PCD_KCFG_DIR)/pcd_autoconf.h $(PCD_KCFG_DIR)/auto.conf
+	@rm -rf $(PCD_ROOT)/include $(PCD_ROOT)/bin
 
 .PHONY: all install check_permissions check_config clean distclean help conf pcd_title menuconfig xconfig defconfig oldconfig

